crates/core/src/network/protobuf.rs

```diff
@@ -74,23 +75,16 @@ pub mod network {
             self.fields.keys().map(String::as_str).collect()
         }
     }
+
     impl Header {
-        #[cfg(feature = "test-util")]
-        pub fn new(
-            nodes_config_version: restate_types::Version,
-            logs_version: Option<restate_types::Version>,
-            schema_version: Option<restate_types::Version>,
-            partition_table_version: Option<restate_types::Version>,
-            in_response_to: Option<u64>,
-        ) -> Self {
-            Self {
-                my_nodes_config_version: Some(nodes_config_version.into()),
-                my_logs_version: logs_version.map(Into::into),
-                my_schema_version: schema_version.map(Into::into),
-                my_partition_table_version: partition_table_version.map(Into::into),
-                msg_id: 0,
-                in_response_to,
-                span_context: None,
+        /// Returns the version of the metadata for the given kind.
+        #[must_use]
```

Comment:
Why is this a `must_use` ?

Reply:
it's a good habit for functions that have no side effects. If the only value of calling a function is its return value, then you'd want the linter to notify them if they aren't using the return value.